style.css

```diff
@@ -0,0 +1,726 @@
+    width: 565px;
+    min-width: 565px;
+    height: 583px;
+}
+
+  .new-collection__title {
+    position: relative;
+    width: 297px;
+    height: 103px;
+    left: 26px;
+    top: 550px;
+    font-weight: bold;
+    font-size: 50px;
+    line-height: 51px;
+}
+
+  .new-collection__title::after {
+    content: "";
+    position: absolute;
+    background-image: url(./img/Vector1.png);
+    width: 35px;
+    height: 20px;
+    left: 145px;
+    top: 115px;
+    cursor: pointer;
+  }
+}
+
+.page2 {
+ width: 100%;
+ height: 900px;
+ background: #fae8e4;
+ position: relative;
+}
+
+.wrap {
+ max-width: 1220px;
+ margin: 0 auto;
+ padding-left: 50px;
+ height: 100%;
+}
+
+.row {
+ display: flex;
+ justify-content: space-between;
+ align-items: center;
+ top: 30px;
+ position: relative;
+}
+
+.bag {
+ padding-left: 90px;
+ background: url(img/whitebag.png) 0 180px no-repeat;
+}
+
+.wrap_part {
+ padding-top: 30px;
+}
+
+.box {
+ background-color: #fff;
+ padding: 40px 65px 44px 55px;
+ max-width: 368px;
+ box-shadow: 0 14px 28px rgba(0, 0, 0, 0.25), 0 10px 10px rgba(0, 0, 0, 0.22);
+}
+
+.clutch__title{
+ font-size: 40px;
+ line-height: 41px;
+ font-family: Montserrat, sans-serif;
+ margin: 0;
+}
+
+.clutch__subtitle {
+ font-size: 16px;
+ line-height: 16px;
+ letter-spacing: 0.07em;
+ opacity: 0.5;
+ font-family: Montserrat, sans-serif;
+ margin-bottom: 55px;
+}
+
+.clutch__description-opacity {
+ opacity: 0.5;
+ margin-bottom: 55px;
+}
+
+.box__color-cost {
+ display: flex;
+ justify-content: space-between;
+ align-items: center;
+}
+
+.circle {
+ display: inline-block;
+ margin-right: 12px;
+ border-radius: 50%;
+ height: 30px;
+ width: 30px;
+}
+
+.circle_purple {
+ background: #dedede;
+}
+
+.circle_pink {
+ background-color: #f29f9f;
+ height: 38px;
+ width: 38px;
+}
+
+.circle_blue {
+ background-color: #a8bff9;
+}
+
+.circle_green {
+ background-color: #90bd8f;
+ margin-right: 0;
+}
+
+.cost {
+ font-size: 40px;
+ line-height: 56px;
+ text-align: right;
+ font-family: Montserrat, sans-serif;
+}
+
+.btn {
+ border: 1.5px solid #000;
+ text-decoration: none;
+ font-family: Montserrat, sans-serif;
+ font-size: 13px;
+ padding: 14px 35px;
+ color: #000;
+ text-transform: uppercase;
+ display: inline-block;
+}
+
+.btn:hover {
+ box-shadow: 5px 5px 5px rgba(0, 0, 0, 0.2);
+}
+
+.btn_cost {
+ margin-right: 15px;
+}
+
+.btn_next {
+ margin-right: 0;
+ margin-top: 40px;
+}
+
+.clutch__other {
+ background: url(img/long_arrow.png) right no-repeat;
+ text-decoration: none;
+ width: 480px;
+ font-size: 13px;
+ line-height: 90px;
+ margin-top: 50px;
+ margin-left: -50px;
+ color: #000;
+ text-transform: uppercase;
+ font-family: Montserrat, sans-serif;
+ position: absolute;
+
+}
+
+.сlutch__other: hover {
+ color: #696969;
+ cursor: pointer;
+ text-shadow: 1px 1px 2px #dcdcdc;
+}
+
+@media (max-width: 1300px) {
+ .wrap {
+   padding-right: 30px;
+   padding-left: 0;
+ }
+
+ .btn_cost {
+   display: none;
+ }
+
+ .cost {
+   font-size: 30px;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Locc/pullRequest?prid=5986341)